hs_core/tests/api/rest/test_resource_system_meta.py

```diff
@@ -0,0 +1,42 @@
+# import json
+
+from rest_framework import status
+
+from hs_core.hydroshare import resource
+from .base import HSRESTTestCase
+
+
+class TestResourceSystemMetadata(HSRESTTestCase):
```

Comment:
Hydroshare deals with two types of metadata - one is 'science' metadata the other one is 'system' metadata. The REST end point work in this PR relates to 'science' metadata. So it would better to name this class as ```TestResourceScienceMetadata()```.